docker-connector-product/README.md

```diff
@@ -0,0 +1,16 @@
+# Docker Connector
```

Comment:
Hi @andreasbalsiger, 
Could you please help review if the readme file fits the market landscape?
Regards
Linh